tests/test_distro.py

```diff
@@ -1905,6 +1908,12 @@ def test_linux_distribution(self) -> None:
         i = _distro.linux_distribution()
         assert i == ("Ubuntu", "14.04", "Trusty Tahr")
 
+        _distro = distro.LinuxDistribution(
+            include_lsb=False, os_release_file=self.fedora30_os_release
+        )
+        i = _distro.linux_distribution()
+        assert i == ("Fedora", "30", "Thirty")
```

Comment:
I checked back with a Docker container of Fedora 30 and it seems the copy of `/etc/os_release` is no longer up to date. With the data from the live container, we'd get `Container Image` rather than `Thirty` here. I hope that's okay for a "version"? For proof:

```diff
# diff -u0 ./tests/resources/distros/fedora30/etc/os-release{,-docker}
--- ./tests/resources/distros/fedora30/etc/os-release   2022-02-12 16:38:39.255999496 +0100
+++ ./tests/resources/distros/fedora30/etc/os-release-docker    2022-02-12 16:52:56.252442968 +0100
@@ -2 +2 @@
-VERSION="30 (Thirty)"
+VERSION="30 (Container Image)"
@@ -7 +7 @@
-PRETTY_NAME="Fedora 30 (Thirty)"
+PRETTY_NAME="Fedora 30 (Container Image)"
@@ -12 +12 @@
-DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/30/system-administrators-guide/"
+DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
@@ -19,0 +20,3 @@
+VARIANT="Container Image"
+VARIANT_ID=container
+
```

Maybe Fedora is in the wrong here and not distro, I'm not sure. Feel free to ignore.

Reply:
I'm not sure we should base our tests on information shipped in container rootfs, they are known not to represent official images (remember #262 !) :smile:

Reply:
If people live in both worlds, we need to support both. I don't know. I thought I should mention it, maybe it doesn't have to be focus here, your call.

Reply:
Yes sir, we do.
From my PoV, we need to support all existing formats, but not their content, as long as they respect the format.
This os-release file you found in Fedora Docker container is indeed different, but does not contain unsupported stuff. So I don't think we should address this here :wink: